packages/task/src/browser/task-service.ts

```diff
@@ -54,12 +58,28 @@ import { ProblemMatcherRegistry } from './task-problem-matcher-registry';
 import { TaskSchemaUpdater } from './task-schema-updater';
 import { TaskConfigurationManager } from './task-configuration-manager';
 import { PROBLEMS_WIDGET_ID, ProblemWidget } from '@theia/markers/lib/browser/problem/problem-widget';
+import { TaskNode } from './task-node';
 
 export interface QuickPickProblemMatcherItem {
     problemMatchers: NamedProblemMatcher[] | undefined;
     learnMore?: boolean;
 }
 
+interface TaskGraphNode {
+    taskConfiguration: TaskConfiguration;
+    node: TaskNode;
+}
+
+export enum TasksEndedTypes {
```

Comment:
Any reason we name it `TasksEndedTypes` and not `TaskEndedTypes` similarly to other interfaces?

Reply:
fixed, done